packages/init/src/tasks/checkReactNative.ts

```diff
@@ -0,0 +1,34 @@
+import fs from 'fs';
+import path from 'path';
+import semver from 'semver';
+
+import logger from '../utils/logger.js';
+
+/**
+ * Checks whether react-native is installed and returns it's version
+ *
+ * @param cwd
+ * @returns React-Native version
+ */
+export default function checkReactNative(rootDir: string): string {
+  const packageJsonPath = path.join(rootDir, 'package.json');
+  const packageJson = JSON.parse(fs.readFileSync(packageJsonPath, 'utf-8'));
```

Comment:
maybe it's also worth checking if package.json exists first, and wrapping JSON.parse in try/catch statement since it may throw an error if JSON file is malformed

Reply:
This will be caught by try/catch clause in the `index.js`, displaying the error from `fs.readFileSync` there.